examples/mnist/Instructions.md

```diff
@@ -0,0 +1,27 @@
+<p align="center"><b>Setup, build and run tiny-dnn on various operating systems</b><p align="center">
+
+<b>tiny-dnn</b> is a C++11 implementation of deep learning. It is suitable for deep learning on limited computational resource, embedded systems and IoT devices.
+
+
```

Comment:
@karandeepdps You can also specify language name for syntax highlighting of code fencing markdown.
After first triple backticks, you may specify it. Like this ```bash

And you can close the code block with triple backticks. ```